internal/ghworkflow/workflow_release.go

```diff
@@ -58,4 +71,10 @@ func releaseWorkflow(cfg core.Configuration) {
 	w.Jobs = map[string]job{"release": j}
 
 	writeWorkflowToFile(w)
+	isInternal := strings.HasPrefix(cfg.Metadata.URL, "https://github.wdf.sap.corp")
```

Comment:
I would like this to not hardcode a specific GHE instance. Can we have it match on a domain that is not `github.com`, and then use the hostname to construct the URLs that appear in the goreleaser config?